dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/handlers/AndroidLogsHandler.kt

```diff
@@ -0,0 +1,40 @@
+package com.datadog.android.tracing.internal.handlers
+
+import com.datadog.android.log.Logger
+import datadog.opentracing.LogsHandler
+
+internal class AndroidLogsHandler(
+    val logger: Logger
+) : LogsHandler {
+
+    // region Span
+    override fun log(timestampMicroseconds: Long, fields: Map<String, *>) {
+        // not sure that we should do something with the timestamp here ???
+        logFields(fields)
+    }
+
+    override fun log(event: String) {
+        logger.v(event)
+    }
+
+    override fun log(timestampMicroseconds: Long, event: String) {
+        // not sure that we should do something with the timestamp here
+        logger.v(event)
+    }
+
+    override fun log(map: Map<String, *>) {
+        logFields(map)
+    }
+
+    // endregion
+
+    // region Internal
+
+    private fun logFields(fields: Map<String, *>) {
+        fields.forEach {
+            logger.addTag(it.key, it.value.toString())
```

Comment:
Here we do only add tags to the `logger` for given span. Shouldn't we replace previously set tags with the new ones?

Reply:
Good call...I was not actually sure what to do with this but @xgouchet suggestion is the right one..which will cover also your comment ;)